packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.mjml

```diff
@@ -2,4 +2,26 @@
   # License, v. 2.0. If a copy of the MPL was not distributed with this
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
-<%- include('/partials/cadReminder/index.mjml') %>
+<mj-section>
```

Comment:
I decided to inline the cadReminder partial. The extra indirection and variables for one additional email doesn't seem worth it.